nixos/modules/services/misc/airsonic.nix

```diff
@@ -143,11 +143,11 @@ in {
       };
     };
 
-    users.users.airsonic = {
-      description = "Airsonic service user";
-      name = cfg.user;
-      home = cfg.home;
-      createHome = true;
-    };
+    users.users = optionalAttrs (cfg.user == "airsonic") (singleton
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/63103.

Specifically the `singleton` usage should be dropped.